client/src/components/accountInfo/accountInfo.jsx

```diff
@@ -50,18 +57,55 @@ function AccountInfo() {
       },
    };
 
+   const handleSubmit = async (event) => {
+      event.preventDefault();
+  
+   
+      try {
+         console.log(userInfo);
+        // Send an axios post request with the updated information
+        const response = await axios.post('/api/updateUserInfo', userInfo);
+        console.log('User info updated:', response);
+        props.updateUserInfo();
+      } catch (error) {
```

Comment:
consider letting the user know there's an error, an error toast, paragraph element